lib/googleauth/application_default.rb

```diff
@@ -21,7 +21,7 @@ module Google
   module Auth
     NOT_FOUND_ERROR = <<~ERROR_MESSAGE.freeze
       Could not load the default credentials. Browse to
-      https://developers.google.com/accounts/docs/application-default-credentials
+      https://cloud.google.com/docs/authentication/provide-credentials-adc
```

Comment:
Per (1.2.3) both mesage and link has to be updated. the "external" link does redirect and is expected, it allows tech writer to make changes later: 

Your credentials were not found. To set up Application Default Credentials for your environment, see https://cloud.google.com/docs/authentication/external/set-up-adc